### ISSUE TITLE
Fetching remote lists

### ISSUE THREAD

**Issue by oragegu** [bookwyrm-social/bookwyrm#1547]
**Is your feature request related to a problem? Please describe.**
For the moment there is no way to search or fetch some lists from remote servers

**Describe the solution you'd like**
Allowing fetching remote lists with searching by url.

**A Scenario to illustrate the situation**
Say there is an user A in bookwyrm.social who was curating a list a long time ago. Then there is a new instance B who joined the fediverse only recently. 

An user C from this new instance B saw on mastodon a list published by A. C has the name and the public url of the list, but there is no way for C to browse and bookmark this list from instance B.

**Comment by mouse-reeve** [comment]
Yes this has been something I've been really really meaning to do for ages, thanks for reminding me.

**Comment by oragegu** [comment]
> Related to #799

Should I close this issue given that there is already an issue published with a much clearer description?
